mosquitto/2.0.15.patch

```diff
@@ -0,0 +1,148 @@
+From a840ddcf4c7c75d5dc28a8c154b51fad9a39b29e Mon Sep 17 00:00:00 2001
+From: Eric Blankenhorn <eric@wolfssl.com>
+Date: Mon, 23 Jan 2023 08:20:51 -0600
+Subject: [PATCH] Support wolfssl instead of openssl
+
+Changes:
+- `config.mk`: add the `WITH_TLS=wolfssl` option
+- The`wolfssl/options.h` header is included by defining the `EXTERNAL_OPTS_OPENVPN` macro
```

Comment:
Is it a significant effort/change to manually include options.h throughout mosquitto?

Reply:
This follows the example of other OSP projects:
bind9
git
krb5
libimobiledevice
net-snmp

Reply:
I'm to blame for this trend.